### PR TITLE
Fix functional tests by aligning versions of different packages

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**

### CODE REVIEW

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config [hunk 1]
```diff
@@ -47,26 +47,6 @@
         <Add>127.0.0.1</Add>
       </ExcludeComponentCorrelationHttpHeadersOnDomains>
     </Add>
-    <Add Type="Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.PerformanceCollectorModule, Microsoft.AI.PerfCounterCollector">
```
Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config]: Removed intentionally?

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj [hunk 2]
```diff
@@ -46,24 +46,27 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="Microsoft.Data.Edm, Version=5.6.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\..\..\..\..\packages\Microsoft.Data.Edm.5.6.0\lib\net45\Microsoft.Data.Edm.dll</HintPath>
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\..\..\packages\Microsoft.Data.Edm.5.6.2\lib\net40\Microsoft.Data.Edm.dll</HintPath>
```
Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj]: why net40?

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj [hunk 3]
```diff
@@ -104,7 +106,7 @@
       <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.ScriptManager.WebForms.5.0.0\lib\net45\Microsoft.ScriptManager.WebForms.dll</HintPath>
     </Reference>
     <Reference Include="System.Web.Optimization, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
-      <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.Web.Optimization.1.1.3\lib\net45\System.Web.Optimization.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.Web.Optimization.1.1.3\lib\net40\System.Web.Optimization.dll</HintPath>
```
Comment (cijothomas) [inline Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj]: why net40?

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config (hunk 1 above)
Comment (SergeyKanzhelev) [reply Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config]: there were no reference on performance counter project anyway. So I figured no need to throw extra exceptions on loading

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj (hunk 2 above)
Comment (SergeyKanzhelev) [reply Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj]: I mistakenly changed it to net45 in previous PR. This package only has net40 and PCL targets

File: Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj (hunk 3 above)
Comment (SergeyKanzhelev) [reply Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj]: Same as above - reverted the previous mistake

**Pull request merged**